src/main/java/com/easypost/easyvcr/requestelements/HttpVersion.java

```diff
@@ -0,0 +1,65 @@
+package com.easypost.easyvcr.requestelements;
+
+import org.apache.http.ProtocolVersion;
+
+/**
+ * Represents an HTTP version.
+ */
+public final class HttpVersion {
+    /**
+     * HTTP protocol string.
+     */
+    private final String protocol;
+
+    /**
+     * HTTP minor version number.
+     */
+    private int minor = 0;
+
+    /**
+     * HTTP major version number.
+     */
+    private int major = 0;
+
+    /**
+     * Constructs a new HTTP version.
+     *
+     * @param version the HTTP version string
+     */
+    public HttpVersion(String version) {
+        this.protocol = version;
+    }
+
+    /**
+     * Constructs a new HTTP version.
+     *
+     * @param version the HTTP ProtocolVersion
+     */
+    public HttpVersion(ProtocolVersion version) {
```

Comment:
What is this used for?

Reply:
`HttpURLConnection` stores its HTTP version info in a class called `ProtocolVersion`. This is to take that `ProtocolVersion` and convert it to our EasyVCR HttpVersion class.